packages/button/mwc-button-base.ts

```diff
@@ -15,9 +15,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import '@material/mwc-icon/mwc-icon';
-import '@material/mwc-ripple/mwc-ripple';
```

Comment:
Our style guide recommends importing directly from files instead of file directory bundles (avoids copybara directory name change issues with Sass), so the opposite should be the case here. If there are any instances of `import '@material/mwc-ripple'`, they should point to `import '@material/mwc-ripple/mwc-ripple'`

Reply:
Done.